### ISSUE TITLE
incorrect aria-level on headings?

### ISSUE THREAD

**Issue by stevefaulkner** [speced/respec#308]
looking at http://rawgit.com/w3c/alt-techniques/master/index.html I see aria-level=1 on headings inside a &lt;body>&lt;section>
these should be aria-level=2 and correct elements h2 are being used. 
example:
&lt;section id="toc">  
&lt;h2 id="h2_toc" class="introductory" aria-level="1" role="heading">&lt;/h2>
If the correct headings are used for the nesting level, there should be no need to apply aria-level and role=heading.
so it looks like ARIA is being unnecessarily added and breaking semantics in process
